tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -2016,6 +2023,7 @@ public function testDuplicateDeployment(): void
         $this->assertGreaterThan(0, $deployment['body']['sourceSize']);
         $this->assertEquals(0, $deployment['body']['buildSize']);
         $this->assertEquals($deployment['body']['sourceSize'], $deployment['body']['totalSize']);
+        $this->assertEquals('cli', $deployment['body']['type']);
```

Comment:
Let's find any place where we do createDuplicateDeployment again (another place), and there let's use helper, and ensure type is manual.

If we dont have such a scenario, let's create it somewhere around here